airbyte-webapp/src/pages/destination/DestinationSettingsPage/DestinationSettingsPage.tsx

```diff
@@ -57,7 +69,7 @@ export const DestinationSettingsPage: React.FC = () => {
         connector={destination}
         onSubmit={onSubmitForm}
       />
-      <DeleteBlock type="destination" onDelete={onDelete} />
+      <DeleteBlock extraModal={extraModal} type="destination" onDelete={onDelete} />
```

Comment:
Hm... Can we replace `extraModal` and `extra` prop name with something more descriptive, like "additionalText", or "additionalInfo"?

Reply:
I suggested `addtionalContent` above

Reply:
Could this be `footer`?

Reply:
The modal's footer is where the button go

Reply:
+1 to `addtionalContent`